demos/browser/app/meeting/meeting.ts

```diff
@@ -491,6 +492,21 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
     });
   }
 
+  setUpVideoTileElementResizer(): void {
+    for (let i = 0; i <= DemoTileOrganizer.MAX_TILES; i++) {
+      const videoElem = document.getElementById(`video-${i}`) as HTMLVideoElement;
+      videoElem.onresize = () => {
+        if (videoElem.videoHeight > videoElem.videoWidth) {
+          // portrait mode
+          this.log(`video-${i} changed to portrait mode resolution ${videoElem.videoWidth}x${videoElem.videoHeight}`);
+          videoElem.style.objectFit = 'contain';
+        } else {
+          videoElem.style.objectFit = 'cover';
```

Comment:
Shouldn't we also compare to the aspect ratio of the `parentElement`? We don't want to apply `cover` when rendering a 16:9 video in a 9:16 (or 4:3) container, because the visible area of the video will pixelate due to stretching and video will focus in the absolute center commonly missing the user's intended focus area.

Reply:
The PR is mainly to show one way of obtaining intrinsic video resolution from HTMLVideoElement. And I agree with your comments, depending on the layout, developers need to handle cropping differently based on various intrinsic video resolutions.